controllers/compaction/metrics.go

```diff
@@ -53,8 +56,15 @@ var (
 		prometheus.HistogramOpts{
 			Namespace: namespaceEtcdDruid,
 			Subsystem: subsystemCompaction,
-			Name:      "job_duration_seconds",
-			Help:      "Total time taken in seconds to finish a running compaction job.",
+			Buckets: []float64{baseDuration.Seconds(),
+				5 * baseDuration.Seconds(),
+				10 * baseDuration.Seconds(),
+				15 * baseDuration.Seconds(),
+				20 * baseDuration.Seconds(),
+				30 * baseDuration.Seconds(),
+			},
```

Comment:
Just a nit. It's easier for readability if the `Buckets` field comes towards the end of the struct, so that the name and purpose of the metric is stated first, followed by other parameters such as buckets for histogram.

Reply:
Done